cloud_run_service_pubsub/main.tf

```diff
@@ -34,6 +34,7 @@ resource "google_cloud_run_service_iam_binding" "binding" {
 
 # [START cloudrun_service_pubsub_token_permissions]
 resource "google_project_iam_binding" "project" {
+  project = "PROJECT_ID"  # project - (Required) The project id of the target project. This is not inferred from the provider.
```

Comment:
Let's use a data source instead which can infer from the provider. This will help us with testing incase we choose to add this in the future. Example:
https://github.com/terraform-google-modules/terraform-docs-samples/blob/893d53ec4e0d03503752327ab44998a6d4ec9aaa/os_login/main.tf#L41-L45